roachtest/awsdms: run once a week instead

otan opened the pull request:
Save a bit of mad dosh by running awsdms once a weekly instead of daily. We don't need this tested every week.

Epic: None

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/108160)
<!-- Reviewable:end -->

smg260 commented on pkg/cmd/roachtest/tests/awsdms.go:
```diff
@@ -192,7 +192,7 @@ func registerAWSDMS(r registry.Registry) {
 		Cluster: r.MakeClusterSpec(1),
 		Leases:  registry.MetamorphicLeases,
-		Tags:    registry.Tags(`default`, `awsdms`, `aws`),
+		Tags:    registry.Tags(`default`, `awsdms`, `aws`, `weekly`),
```
I think the only 2 tags required here are `weekly` and `aws-weekly`. The former gets special treatment in roachtest [1], and the latter is required to actually run it in the weekly aws job.

[1] - [test_registry.go](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/test_registry.go#L149)
[2] - [roachtest_weekly_aws_impl.sh](https://github.com/cockroachdb/cockroach/blob/master/build/teamcity/cockroach/nightlies/roachtest_weekly_aws_impl.sh#L19)

otan replied:
thanks! amended.

otan committed:
roachtest/awsdms: run once a week instead

Save a bit of mad dosh by running awsdms once a weekly instead of daily.
We don't need this tested every week.

Epic: None

Release note: None

otan commented:
bors r=Jeremyyang920
